akka-javasdk/src/main/java/akka/javasdk/agent/SessionMessage.java

```diff
@@ -20,7 +20,7 @@ public sealed interface SessionMessage {
 
   int size();
 
-  record UserMessage(String text, int tokens) implements SessionMessage {
+  record UserMessage(long timestamp, String text, int tokens) implements SessionMessage {
```

Comment:
is there a reason why we don't use `Instant` as timestamp? as far as I know we have support for Instant in views

Reply:
We have support for it, but I faced an issue with Instant lately, it's on my list to debug it, I think that long is just a safe choice, but I would let @efgpinto comment on that.